torch/onnx/experimental/fonnx/v1/custom_operators.py

```diff
@@ -0,0 +1,53 @@
+import onnxscript
+import torch
+from contextlib import contextmanager
+
+
+def register_custom_operator(
+    operator: torch._ops.OpOverload,
+    opset_version: int,
+    symbolic_fn: onnxscript.OnnxFunction,
+):
+    """Register `symbolic_fn` onnx function for custom `operator` and  opset version.
+
+    TODO: Check whether we need to add `Callable` at `operator`'s type annotation
+
+    Args:
+        operator: Unique operator identifier for registration.
+        symbolic_fn: Function that adds an `operator` representation in the ONNX graph.
+        opset_version (int): The ONNX opset version in which to register.
+    """
+    ...
+
+
+def unregister_custom_operator(operator: torch._ops.OpOverload,
+                               opset_version: int):
+    """Unregister `operator` from ONNX Registry.
+
+    TODO: Check whether we need to add `Callable` at `operator`'s type annotation
```

Comment:
Add to comment "Start with more restricted set of only `torch._ops.OpOverload`".